tests/test_app.py

```diff
@@ -18,8 +18,8 @@ def token():
 
 @pytest.fixture
 def client():
-    app.config["ENROLLMENT"] = enrollment()
-    app.config["TOKEN"] = '123'#token()
+    app.config['ENROLLMENT'] = enrollment()
+    app.config['TOKEN'] = '123'#token()
```

Comment:
done in https://github.com/blue-yonder/azure-cost-mon/commit/6462aed9ad60d49880ff9812a11c640a8e48751f